### PR TITLE
Update tllbarbarus.bos and tllcaps speed flak

### PR DESCRIPTION
**Pull request by skymyj**

### CODE REVIEW

**Comment by nixtux** [comment]
Did you not read what I posted in the other thread? With a reload of 0.75 a turnrate of 480 should be sufficient.

**Comment by skymyj** [comment]
ok i will change

**Comment by skymyj** [comment]
Done

File: scripts/tllcaps.bos [hunk 1]
```diff
@@ -117,7 +117,7 @@ Restore5()
 Restore6()
 {
 	sleep restore_delay/2;
-	turn turret8 to y-axis <0.000000> speed <100.000000>;
+	turn turret8 to y-axis <180.000000> speed <100.000000>;
```
Comment (skymyj) [inline scripts/tllcaps.bos]: bad restore direction

**Pull request merged**